client/stylesheets/_toCat.scss

```diff
@@ -4365,7 +4365,11 @@ On click the images will be displayed at normal size to complete the effect
     margin: 5px;
     padding: 15px 10px;
     color: $color-spacered;
-    outline: 2px solid red;
+
+    // Nice outline
+    border-top: 2px solid red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces